ElementX/Sources/Services/Room/RoomProxy.swift

```diff
@@ -218,6 +218,10 @@ class RoomProxy: RoomProxyProtocol {
             }
         }
     }
+    
+    func sendImage(body: String, url: URL) async -> Result<Void, RoomProxyError> {
```

Comment:
Should probably handle this Rust side using the last filename from the URL for the body as in the spec.
```suggestion
    func sendImage(url: URL) async -> Result<Void, RoomProxyError> {
```

Reply:
Fair point, I wasn't sure what we were going to do with this from the UI side so I just left it in